Source/Scene/Cesium3DTileset.js

```diff
@@ -497,6 +522,63 @@ define([
         }).otherwise(function(error) {
             that._readyPromise.reject(error);
         });
+
+        /**
+         * Determines if level-of-detail skipping optimization should be used.
+         *
+         * @type {Boolean}
+         * @default true
+         */
+        this.skipLODs = defaultValue(options.skipLODs, true);
+
+        /**
+         * This is a multiplier defining the minumum screen space error to skip.
+         * If current tile has screen space error of 100, no tiles will be loaded unless they
+         * are leaves or have a screen space error <= 100 / skipSSEFactor.
+         *
+         * Only used when tileset.skipLODs === true.
+         *
+         * @type {Number}
+         * @default 10
+         */
+        this._skipSSEFactor = defaultValue(options.skipSSEFactor, 10);
+
+        /**
+         * This is a constant defining the minumum number of levels skip.
+         * If current tile is level 1, no tiles will be loaded unless they
+         * are at level greater than 2.
+         *
+         * Only used when tileset.skipLODs === true.
+         *
+         * @type {Number}
+         * @default 1
+         */
+        this._skipLevels = defaultValue(options.skipLevels, 1);
+
+        /**
+         * Determines whether low memory level-of-detail skipping should be used.
+         * When true, only tiles that meet the maximum screen space error will ever be downloaded.
+         *
+         * Only used when tileset.skipLODs === true.
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.lowMemory = defaultValue(options.lowMemory, false);
+
+        /**
+         * Determines whether sibling tiles should be loaded when skipping levels-of-detail.
```

Comment:
Focus more on why something happens, not just what happens.  Why would a user want this?  To prefect for when the viewer turns left/right?  So there is less of a pop during refinement?  In practice, will users want to change this option?